src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -77,6 +77,16 @@ class ReportController(
         return reports.map { name -> ReportWithDate(name, versionedReports.find { it.name == name }?.date) }
     }
 
+    fun getParameterRunReports(): List<Parameters>
```

Comment:
Would be nice to have a simple test for this, so check that parameters and any default values are correctly deserialised

Reply:
There is a test for the controller method already.